lalrpop/src/tok/mod.rs

```diff
@@ -548,7 +548,13 @@ impl<'input> Tokenizer<'input> {
         self.lookahead.and_then( |(_, c)| {
             if c == '\\' {
                 // escape after `'` => it had to be character literal token, consume
-                self.take_until_and_consume_terminating_character(|c:char| { c == '\'' })
```

Comment:
Wouldn't it be better to call `bump` and then call `take_until_and_consume_terminating_character` as before?

Reply:
Ah, good point.